ts/indicators/indicators.src.ts

```diff
@@ -537,13 +538,16 @@ seriesType<Highcharts.SMAIndicator>(
                 yData: yData
             } as Highcharts.IndicatorValuesObject<TLinkedSeries>;
         },
-        destroy: function (this: Highcharts.SMAIndicator): void {
+        destroy: function (
+            this: Highcharts.SMAIndicator,
+            keepEvents?: boolean
+        ): void {
             this.dataEventsToUnbind.forEach(function (
                 unbinder: Function
             ): void {
                 unbinder();
             });
-            Series.prototype.destroy.call(this);
+            Series.prototype.destroy.call(this, keepEvents);
```

Comment:
Wouldn't it be better to use `Series.prototype.destroy.call(this, ...arguments);` or `Series.prototype.destroy.apply(this, arguments);`? `keepEvents` wasn't always there, so if another argument will be added we will need to fix this again..